### PR TITLE
update /account | /address | /aeuis | /applications

### PR DESCRIPTION
**Pull request by maksymmateiuk**

### CODE REVIEW

File: site/docs/numbers/applications.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+#### For ServiceType of Messaging-V2:
+* MsgCallbackUrl - is deprecated, but required if InboundCallbackUrl is not specified. Updates both InboundCallbackUrl and OutboundCallbackurl
+* InboundCallbackUrl - required for messaging.
+* OutboundCallbackUrl - optional for messaging.
+* InboundCallbackCreds - optional for messaging. Corresponds to InboundCallbackUrl.
+* OutboundCallbackCreds - optional for messaging. Corresponds to OutboundCallbackUrl.
+* RequestedCallbackTypes - optional for messaging. List containing callback types: message-delivered, message-sending, message-failed.
+
+#### For ServiceType of Voice-V2:
```
Comment (mrmeyers99) [inline site/docs/numbers/applications.mdx:39]: We should probably drop the V2 here since V1 is gone

File: site/docs/numbers/applications.mdx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        color: '#fff',
+        padding: '0.2rem',
+      }}>
+      {children}
+    </span>
+  );
+
+The application is a resource that provides customers with the ability to control messaging (SMS and MMS) and voice using HTTP signaling. This resource identifies the callback information used to notify the customer about voice and messaging events. The application is represented by an application id. The API calls that are used to manage the details of a customer-defined application are accessed through the /accounts/{accountId}/applications resource.
+
+#### For ServiceType of Messaging-V2:
```
Comment (mrmeyers99) [inline site/docs/numbers/applications.mdx:31]: We should probably drop the V2 here since V1 is gone

Comment (maksymmateiuk) [reply site/docs/numbers/applications.mdx]: done.

File: site/docs/numbers/applications.mdx (hunk 1 above)
Comment (maksymmateiuk) [reply site/docs/numbers/applications.mdx]: done.

File: site/specs/numbers.json [hunk 3]
```diff
@@ -448,8 +445,7 @@
...
...>15jPWZmXdm</UserId>\n    </OutboundCallbackCreds>\n    <RequestedCallbackTypes>\n      <CallbackType>message-delivered</CallbackType>\n      <CallbackType>message-failed</CallbackType>\n      <CallbackType>message-sending</CallbackType>\n    </RequestedCallbackTypes>\n  </Application>\n</ApplicationProvisioningResponse>\n<!--Voice-V2 application-->\n<ApplicationProvisioningResponse>\n<Application>\n  <ApplicationId>d3e418e9-1833-49c1-b6c7-ca1700f79586</ApplicationId>\n    <ServiceType>Voice-V2</ServiceType>\n    <AppName>v1</AppName>\n    <CallInitiatedCallbackUrl>https://a.com</CallInitiatedCallbackUrl>\n    <CallInitiatedMethod>GET</CallInitiatedMethod>\n    <CallStatusCallbackUrl>https://b.com</CallStatusCallbackUrl>\n    <CallStatusMethod>GET</CallStatusMethod>\n    <CallbackCreds>\n      <UserId>login123</UserId>\n    </CallbackCreds>\n    <CallInitiatedFallbackUrl>https://fallback.com</CallInitiatedFallbackUrl>\n    <CallInitiatedFallbackMethod>POST</CallInitiatedFallbackMethod>\n    <CallInitiatedFallbackCreds>\n      <UserId>login2</UserId>\n    </CallInitiatedFallbackCreds>\n    <CallbackTimeout>20</CallbackTimeout>\n  </Application>\n  </ApplicationProvisioningResponse>"
```
Comment (mrmeyers99) [inline site/specs/numbers.json:448]: It works be good to split this into two examples, one for messaging and one for voice. This xml isn't valid

File: site/specs/numbers.json [hunk 4]
```diff
@@ -497,8 +493,7 @@
...
...kCreds>\n    <InboundCallbackCreds>\n      <UserId>15jPWZmXdm</UserId>\n    </InboundCallbackCreds>\n    <OutboundCallbackCreds>\n      <UserId>15jPWZmXdm</UserId>\n    </OutboundCallbackCreds>\n    <RequestedCallbackTypes>\n      <CallbackType>message-delivered</CallbackType>\n      <CallbackType>message-failed</CallbackType>\n      <CallbackType>message-sending</CallbackType>\n    </RequestedCallbackTypes>\n  </Application>\n</ApplicationProvisioningResponse>\n<!--Voice-V2 application example-->\n<ApplicationProvisioningResponse>\n  <Application>\n    <ApplicationId>e5a9e103-097e-4ec4-87a0-50109fb7b4b1</ApplicationId>\n    <ServiceType>Voice-V2</ServiceType>\n    <AppName>qMmmTT</AppName>\n    <CallInitiatedCallbackUrl>http://example.com</CallInitiatedCallbackUrl>\n    <CallbackCreds>\n      <UserId>15jPWZmXdm</UserId>\n    </CallbackCreds>\n    <CallInitiatedFallbackUrl>https://fallback.com</CallInitiatedFallbackUrl>\n    <CallInitiatedFallbackMethod>POST</CallInitiatedFallbackMethod>\n    <CallInitiatedFallbackCreds>\n      <UserId>login2</UserId>\n    </CallInitiatedFallbackCreds>\n    <CallbackTimeout>20</CallbackTimeout>\n  </Application>\n</ApplicationProvisioningResponse>"
```
Comment (mrmeyers99) [inline site/specs/numbers.json:496]: This example would be good to split up as well

File: site/specs/numbers.json [hunk 5]
```diff
@@ -510,39 +505,38 @@
...
...erviceType>Messaging-V2</ServiceType>\n  <AppName>EBVICs</AppName>\n  <InboundCallbackUrl>http://example.com</InboundCallbackUrl>\n  <OutboundCallbackUrl>http://example2.com</OutboundCallbackUrl>\n  <RequestedCallbackTypes>\n    <CallbackType>message-delivered</CallbackType>\n    <CallbackType>message-failed</CallbackType>\n    <CallbackType>message-sending</CallbackType>\n  </RequestedCallbackTypes>\n  </Application>\n<!--Voice-V2 application-->\n<Application>\n  <ServiceType>Voice-V2</ServiceType>\n  <AppName>v1</AppName>\n  <CallInitiatedCallbackUrl>https://a.com</CallInitiatedCallbackUrl>\n  <CallInitiatedMethod>GET</CallInitiatedMethod>\n  <CallStatusCallbackUrl>https://b.com</CallStatusCallbackUrl>\n  <CallStatusMethod>GET</CallStatusMethod>\n  <CallbackCreds>\n    <UserId>login123</UserId>\n    <Password>password123</Password>\n  </CallbackCreds>\n  <CallInitiatedFallbackUrl>https://fallback.com</CallInitiatedFallbackUrl>\n  <CallInitiatedFallbackMethod>POST</CallInitiatedFallbackMethod>\n  <CallInitiatedFallbackCreds>\n    <UserId>login2</UserId>\n    <Password>password123</Password>\n  </CallInitiatedFallbackCreds>\n  <CallbackTimeout>20</CallbackTimeout>\n </Application>"
```
Comment (mrmeyers99) [inline site/specs/numbers.json:539]: And here

File: site/specs/numbers.json [hunk 6]
```diff
@@ -568,30 +561,30 @@
         "tags": [
           "/accounts"
         ],
```
Comment (mrmeyers99) [inline site/specs/numbers.json:563]: And here

File: site/specs/numbers.json [hunk 7]
```diff
@@ -414,28 +410,29 @@
...
-            "name": "accountId",
-            "description": "",
-            "in": "path",
-            "required": true,
-            "schema": {
-              "type": "string",
-              "default": null
-            },
-            "example": null
+          "name": "accountId",
```
[the remark this answers is not in the document]
Comment (maksymmateiuk) [reply site/specs/numbers.json:418]: my bad copy-paste

**Review by ajrice6713 — CHANGES_REQUESTED** [review]

File: site/specs/numbers.json [hunk 8]
```diff
@@ -55,31 +55,33 @@
         "tags": [
           "/accounts"
         ],
-        "description": "<p>This API call retrieves information about the account indicated by the Account ID.</p>\n",
-        "operationId": "GET /accounts/{accountId}",
+        "description": "<p>The account is the root resource for many of the operations in the Bandwidth Dashboard API.</p> <p>The account is represented by an account id, which is the resource that represents a Bandwidth customer, providing a root resource for all of the customer's attributes and services.</p> The API calls that are used to manage the details of a customer account, and to manage the resources that a Bandwidth customer has access to or control over, are accessed through the /accounts resource.</p> <p>This API call retrieves information about the account indicated by the Account ID.</p>",
```
[the remark this answers is not in the document]
Comment (maksymmateiuk) [reply site/specs/numbers.json:58]: Thank you!

File: site/specs/numbers.json [hunk 9]
```diff
@@ -93,45 +95,45 @@
...
-              "type": "string",
-              "default": null
+              "type": "string"
             },
             "example": "e911"
           },
           {
             "name": "page",
-            "description": "<p>An ID of the first element in a page.  This value will indicate the first value, not the page number, of the initial entry in the page being requested.<br> It can be an id of address. The initial page is tagged with the ID '1'.</p>\n",
+            "description": "An ID of the first element in a page.  This value will indicate the first value, not the page number, of the initial entry in the page being requested.<br> It can be an id of address. The initial page is tagged with the ID '1'.",
```
[the remark this answers is not in the document]
Comment (maksymmateiuk) [reply site/specs/numbers.json:136]: changed.

File: site/docs/numbers/applications.mdx [hunk 10]
```diff
@@ -0,0 +1,68 @@
+---
+id: applications
```
[the remark this answers is not in the document]
Comment (maksymmateiuk) [reply site/docs/numbers/applications.mdx:2]: Didn't see it, thank you!

[the remark this answers is not in the document]
Comment (maksymmateiuk) [reply site/docs/numbers/applications.mdx]: I thought I deleted it. Deleted now.

**Review by ajrice6713 — APPROVED** [review]

**Review by mrmeyers99 — APPROVED** [review]

**Pull request merged**